src/utils.js

```diff
@@ -50,7 +50,13 @@ async function setUpNpmConfig (registry, strictSSL) {
     rollback: false,
     fund: false,
     'strict-ssl': strictSSL,
-    noproxy: 'registry.npmjs.org'
+    noproxy: 'registry.npmjs.org',
+    // https://docs.npmjs.com/cli/v6/using-npm/config#package-lock
+    // By default, `npm install $package` will install `$package` as well
+    // as any dependency defined in package-lock.json that is missing from
+    // node_modules.
+    // Setting to false means `npm install $package` only installs `$package`
+    'package-lock': false
```

Comment:
Is this something we may potentially want to be configurable? 🤔 As a just-in-case option. Not necessarily advertised/documented, but rather if we were to detect some unforseen sideeffect a) a user could still have a workaround and b) we could change the default in saucectl without having to release new runners.

Reply:
Good point, no downsides to making it configurable. 👍

Reply:
@alexplischke Looking at the docs for npm config again, every config option can already be set by environment variables. e.g. `NPM_CONFIG_PACKAGE_LOCK=false npm install $PACKAGE` will "just work" ™️ 

I'm inclined to go that route. It's simpler to change and leverages out of the box npm functionality which makes it easier to understand.